lib/toolchain.rb

```diff
@@ -1,55 +1,52 @@
 # frozen_string_literal: true
 
-TAP_PATH = (HOMEBREW_TAP_DIRECTORY/"MaterializeInc"/"homebrew-crosstools").freeze
+TAP_PATH = (HOMEBREW_TAP_DIRECTORY/"def-"/"homebrew-crosstools").freeze
 
 class Toolchain < Formula
   desc "Appease RuboCop; not used"
   homepage "https://see.above"
 
   DEFCONFIG = <<~EOF
     <%= defconfig %>
-    CT_BINUTILS_GOLD_THREADS=y
-    CT_BINUTILS_LD_WRAPPER=y
-    CT_BINUTILS_LINKER_LD_GOLD=y
-    CT_BINUTILS_PLUGINS=y
-    CT_BINUTILS_V_2_37=y
-    CT_CC_GCC_BUILD_ID=y
-    CT_CC_LANG_CXX=y
-    CT_CONFIG_VERSION="3"
-    CT_EXPERIMENTAL=y
-    CT_FORBID_DOWNLOAD=y
-    CT_GCC_V_11=y
-    CT_GETTEXT_V_0_21=y
-    CT_GLIBC_V_<%= glibc_version.gsub(".", "_") %>=y
-    # CT_GLIBC_ENABLE_OBSOLETE_RPC is not set
-    CT_GMP_V_6_2=y
-    CT_ISL_V_0_24=y
-    CT_KERNEL_LINUX=y
-    CT_LIBC_GLIBC=yCT_DEBUG_CT=y
-    CT_LIBICONV_V_1_16=y
-    CT_LINUX_V_<%= linux_version.gsub(".", "_") %>=y
     CT_LOCAL_TARBALLS_DIR="<%= buildpath %>/tarballs"
     CT_LOCAL_PATCH_DIR="<%= buildpath %>/patches"
     # CT_LOG_PROGRESS_BAR is not set
-    CT_MPC_V_1_2=y
-    CT_MPFR_V_4_1=y
-    CT_NCURSES_V_6_2=y
+    CT_PATCH_ORDER="bundled,local"
     CT_PATCH_BUNDLED_LOCAL=y
+    CT_PATCH_USE_LOCAL=y
     CT_PREFIX_DIR="<%= buildpath %>/mnt/install"
     <% if Context.current.debug? %>
     CT_DEBUG_CT=y
     CT_DEBUG_CT_SAVE_STEPS=y
+    CT_CONFIG_VERSION="3"
+    CT_EXPERIMENTAL=y
+    CT_ALLOW_BUILD_AS_ROOT=y
+    CT_ALLOW_BUILD_AS_ROOT_SURE=y
+    # CT_SAVE_TARBALLS is not set
+    CT_PREFIX_DIR="${CT_PREFIX:-/opt/x-tools}/${CT_HOST:+HOST-${CT_HOST}/}${CT_TARGET}"
+    # CT_LOG_TO_FILE is not set
+    CT_ARCH_ARM=y
+    CT_ARCH_64=y
+    CT_KERNEL_LINUX=y
+    CT_LINUX_V_5_10=y
+    CT_BINUTILS_LINKER_LD_GOLD=y
+    CT_BINUTILS_GOLD_THREADS=y
+    CT_BINUTILS_LD_WRAPPER=y
+    CT_BINUTILS_PLUGINS=y
+    CT_CC_LANG_CXX=y
+    CT_LIBC_GLIBC=y
+    # CT_LOG_PROGRESS_BAR is not set
+    CT_GLIBC_V_2_35=y
+    # CT_COMP_TOOLS_AUTOCONF is not set
+    # CT_COMP_TOOLS_AUTOMAKE is not set
+    CT_CC_GCC_BUILD_ID=y
```

Comment:
Hard to track what's changing here. Can you alphabetically sort these like they were previously so we get a nice diff?